mkdocs-base.yml

```diff
@@ -159,14 +159,14 @@ nav:
         - lru-dump-backup.md
         - throttling-backups.md
         - store-backup-history.md
+        - reduction-in-locks.md
       - Prepare features:
         - dictionary-cache.md
       - Restore features:
         - point-in-time-recovery.md
         - restore-individual-tables.md
       - flush-tables-with-read-lock.md
       - log-enhancements.md
-      - lock-options.md
```

Comment:
If this is a Pro Build feature - should it be in a Pro Build section to highlight that?